test-requirements.txt

```diff
@@ -0,0 +1 @@
+odoo-addon-ddmrp@git+https://github.com/OCA/ddmrp.git@refs/pull/314/head#subdirectory=setup/ddmrp
```

Comment:
Can you rebase and remove test requirements? This should be good to go after this.